src/app.ts

```diff
@@ -78,7 +79,8 @@ export class CeramicAnchorApp {
     this.anchorsSupported =
       this.mode === AppMode.ANCHOR ||
       this.mode === AppMode.BUNDLED ||
-      this.config.anchorControllerEnabled
+      this.mode === AppMode.CONTINUAL_ANCHORING
```

Comment:
Is this a typo? Looks like the `||` is missing at the end of this line.

Reply:
yup!